freqtrade/optimize/__init__.py

```diff
@@ -77,7 +77,7 @@ def download_pairs(datadir, pairs: List[str]) -> bool:
     """For each pairs passed in parameters, download 1 and 5 ticker intervals"""
     for pair in pairs:
         try:
-            for interval in [1, 5]:
+            for interval in [1, 5, 30, 60, 1440]:
```

Comment:
In this case it will make sense this function to download only the ticker expected.